week1.ipynb

```diff
@@ -0,0 +1,176 @@
+    "\n",
+    "where $m$ is a mass in kg, $g$ is the gravitational acceleration in m/s, $h_f$ is the final height of an object in m (above Earth's surface, unless specified otherwise), and $h_0$ is the initial height of an object in m before any work is done on it. Note that the convention for gravity is positive downwards.\n",
+    "\n",
+    "The relationship between work and potential energy is written as follows:\n",
+    "\n",
+    "$\\Delta PE = -W$\n",
+    "\n",
+    "Why is work negative here? This is a sign convention used because an object that has fallen has $\\textit{lost}$ potential energy, so $\\Delta PE$ is negative. Adding the minus sign in front of $W$ ensures that in such a case, the minus signs cancel, and the work done is still postive."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## An example\n",
+    "\n",
+    "Let's imagine an apple falling to the ground from a height of 10 feet. Its mass is 100 g. We will calculate how much work was done on the apple, and in the question below you will find the change in velocity that it experienced during its fall. We will assume the apple was staionary immediately before falling."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# first thing's first, convert units! Convert all to SI units\n",
+    "\n",
+    "height_initial = 10 / 3.3 # 3.3 feet in 1 meter\n",
+    "mass = 100 /1000 # 1000 grams to a kilogram"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create dictionary of constants\n",
+    "\n",
+    "constants = {'mass_of_apple' : mass,\n",
+    "             'gravitational_acceleration' : 9.81,\n",
+    "             'initial_height' : height_initial,\n",
+    "             'final_height' : 0}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# calculate potential energy\n",
+    "\n",
+    "potential_energy = constants['mass_of_apple'] * constants['gravitational_acceleration'] * (constants['final_height'] - constants['initial_height'])\n",
+    "print('potential energy: ', potential_energy, 'kg m^2 / s^2')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# calculate work done, slightly rearranging the equation by moving the minus sign to solve for work\n",
+    "\n",
+    "work_done = -potential_energy\n",
+    "print('work done: ', work_done, 'kg m^2 / s^2')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Questions\n",
+    "\n",
+    "1. Please describe the types of energy held in the following scenarios. Do you see anything problematic in any of them, given only the information we've learned today?\n",
+    "- A box sitting stationary on a table\n",
```

Comment:
I think you might have to explain a reference frame more explicitly to ask this question. I imply it in the next one though!